scenes/globals/game_state/game_state.gd

```diff
@@ -24,6 +24,12 @@ const QUEST_SPAWNPOINT_KEY := "current_spawn_point"
 const GLOBAL_SECTION := "global"
 const GLOBAL_INCORPORATING_THREADS_KEY := "incorporating_threads"
 
+## Scenes to skip from saving.
+const TRANSIENT_SCENES := [
```

Comment:
How about the splash?

Reply:
Ah. And in fact we never *switch* to the main scene, so `set_scene` is never called for it, so I think it will never be saved. And if it is saved it's harmless enough.